src/DataProtection/Cryptography.KeyDerivation/ref/Microsoft.AspNetCore.Cryptography.KeyDerivation.csproj

```diff
@@ -8,7 +8,7 @@
     <Reference Include="Microsoft.AspNetCore.Cryptography.Internal"  />
   </ItemGroup>
 <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
```

Comment:
@HaoK do you know why this is netcoreapp2.0?

Reply:
It might just be a mistake, I think it can just target netstandard20 like the other core dataprotection packages cc @blowdart

Reply:
What about https://github.com/aspnet/AspNetCore/blob/master/src/DataProtection/Cryptography.KeyDerivation/src/PBKDF2/NetCorePbkdf2Provider.cs?

Reply:
Ah right, sorry I forgot that DataProtection does need cross compiling here due to needing to use netcoreapp API improvements since we are using a netcoreapp only DeriveBytes API.

So the cross compiling was intentional, sorry for the confusion

Reply:
Could the project instead be cross-compiled w/ 'netcoreapp3.1'?

Reply:
Sure it probably should be whatever the latest netcoreapp version is